### ISSUE TITLE
Node agent warns about Gluster on Tendrl node

### ISSUE THREAD

**Issue by japplewhite** [Tendrl/node-agent#458]
The node agent running on the Tendrl node warns as follows.

"May 19 16:43:04 digitalocean journal: 2017-05-19 16:43:04.233069+00:00 - node_agent - /usr/lib/python2.7/site-packages/tendrl/node_agent/discovery/sds/plugins/discover_gluster_sds_details.py:25 - _derive_cluster_id - ERROR - Could not detect SDS:Gluster installation"

Because this is the Tendrl node no SDS is expected. Nor do we want Ceph users seeing these kind of messages if it's not in play. 

We should suppress the warning (and the task) on the Tendrl node. It probably consumes unneeded resources and creates etcd entries that are not needed.

Perhaps the node agent on Tendrl should have a tag in the conf file that overrides this?

**Comment by japplewhite** [comment]
perhaps the node agent on Tendrl should have a tag in the conf file that overrides this?

**Comment by r0h4n** [comment]
@japplewhite The message has been given a "debug" priority , it wont be visible in etcd.

Regarding SDS detection, we can turn it off completely if a node becomes part of an sds cluster managed by Tendrl.

If the node is not part of any tendrl managed cluster, then we should always run sds detection on that node?

**Comment by japplewhite** [comment]
@r0h4n yes we want the cluster detection process (and the process to add bare nodes to Tendrl) to be "greedy" - in other words I would rather see an indication that we tried to import a cluster and it failed for x reason than simply a bare ui - that frustrates the user. I would also rather see a cluster or node show up with partial information and a problem indicator than nothing, especially if we can guide the user to fix something simple that is a problem. My concern around this one was that from my perspective all I had done is import bare nodes - no cluster creation had started and so from that workflow it doesn't make sense to see that a cluster detection failed in the logs. It's ok that we tried and failed, and I assume we would also try to detect a ceph cluster. So if there is a job that runs that tries to detect either type it would be fine to show that we tried and failed to detect any type of storage cluster/pool.
